### PR TITLE
Address `OS.request_permissions()` bug when a non-platform permission is included

### LINKED ISSUE #102: Enabling hand tracking permission breaks OS.request_permissions()
It appears that the presence of the hand tracking permission inside the AndroidManifest.xml throws a NameNotFound exception at
https://github.com/godotengine/godot/blob/7899b3e734e5c64304e8ffb56293fb4b1912b547/platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java#L117
and
https://github.com/godotengine/godot/blob/7899b3e734e5c64304e8ffb56293fb4b1912b547/platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java#L157

This leads to not triggering the permission request dialog and also always returns an empty array for the list of permissions.

A temp. workaround is to continue the loop at these points even when the exception is thrown instead of returning an empty list/false but I don't know the implications of this change.

### PR DESCRIPTION
**Pull request by m4gr3d**
Fixes https://github.com/GodotVR/godot_oculus_mobile/issues/102

### CODE REVIEW

**Comment by m4gr3d** [comment]
cc @NeoSpark314

File: platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java [hunk 1]
```diff
@@ -36,6 +36,8 @@
 import android.content.pm.PermissionInfo;
 import android.os.Build;
 import android.support.v4.content.ContextCompat;
+import android.util.Log;
+
```
Comment (akien-mga) [inline platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java:40]: `clang-format` doesn't like this it seems.

Comment (akien-mga) [reply platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java:40]: Note that if we want to enforce this kind of separation, we can do it in `.clang-format` with `IncludeCategories`.

Comment (akien-mga) [reply platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java:40]: Actually `IncludeCategories` doesn't affect Java, but there's `JavaImportGroups`.

**Comment by akien-mga** [comment]
I force pushed an amend to fix the include style as I want to merge this now and cherry-pick for 3.2.2 beta 2.

See #38474 for a proposal to improve the Java import grouping.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**